server/api/db/sqldb/db.py

```diff
@@ -2846,7 +2846,42 @@ def _calculate_schedules_counters(
         next_day = datetime.now(timezone.utc) + timedelta(hours=24)
 
         schedules_pending_count_per_project = (
-            session.query(Schedule.project, Schedule.name, Schedule.Label)
+            session.query(
+                Schedule.project,
+                Schedule.name,
+                # The logic here is the following:
+                # If the schedule has a label with the name "workflow" then we take the value of that label
+                # name. Otherwise, we take the value of the label with the name "kind".
+                # The reason for that is that for schedule workflow we have both workflow label and kind label
+                # with job, and on schedule job we have only kind label with job and because of that we first
+                # want to check for workflow label and if it doesn't exist then we take the kind label.
+                func.coalesce(
+                    func.max(
```

Comment:
IIUC how everything works then the `func.max` is not needed right?

Reply:
No, we still need the max because on workflow we can get both kind and workflow as label name(because of the group by) and we want to get the workflow and only if it doesn't exists get the kind